talentsearch/resources/js/components/request/CreateRequest.tsx

```diff
@@ -0,0 +1,303 @@
+  ): CreatePoolCandidateSearchRequestInput => ({
+    ...values,
+    department: { connect: values.department },
+  });
+
+  const onSubmit: SubmitHandler<FormValues> = async (data) => {
+    return handleCreatePoolCandidateSearchRequest(formValuesToSubmitData(data))
+      .then(() => {
+        navigate(searchPath());
+        toast.success(
+          intl.formatMessage({
+            defaultMessage: "Request created successfully!",
+            description:
+              "Message displayed to user after a pool candidate request is created successfully.",
+          }),
+        );
+      })
+      .catch(() => {
+        toast.error(
+          intl.formatMessage({
+            defaultMessage: "Error: creating request failed",
+            description:
+              "Message displayed to user after a pool candidate request fails to get created.",
+          }),
+        );
+      });
+  };
+
+  const departmentOptions: Option<string>[] = departments.map(
+    ({ id, name }) => ({
+      value: id,
+      label: name[locale] ?? "Error: department name not loaded",
+    }),
+  );
+
+  // TODO: Replace fake data with data from api
+  const poolCandidateFilter: PoolCandidateFilter =
+    fakePoolCandidateFilters()[0] as PoolCandidateFilter;
+  const classifications: string[] | undefined =
+    poolCandidateFilter.classifications?.map(
+      (classification) =>
+        `${classification?.group.toLocaleUpperCase()}-0${
+          classification?.level
+        }`,
+    );
+  const educationLevel: string | undefined = poolCandidateFilter.hasDiploma
+    ? "Required diploma from post-secondary institution"
+    : "Can accept a combination of work experience and education";
+  const employmentEquity: string[] | undefined = ["Woman", "Visible Minority"];
+  const operationalRequirements: string[] | undefined =
+    poolCandidateFilter.operationalRequirements?.map(
+      (operationalRequirement) =>
+        operationalRequirement?.name.en || "operational requirement",
+    );
+  const skills: string[] | undefined = poolCandidateFilter.cmoAssets?.map(
+    (cmoAsset) => cmoAsset?.name.en || "cmo asset",
+  );
+  const typeOfOpportunity = "Indeterminate position";
+  const workLocation: string[] = poolCandidateFilter.workRegions as string[];
+
+  return (
+    <div>
+      <h2 data-h2-margin="b(top, none)">
+        {intl.formatMessage({
+          defaultMessage: "Request Form",
+          description: "Heading for request form.",
+        })}
+      </h2>
+      <p>
+        {intl.formatMessage({
+          defaultMessage:
+            "To submit a request, please provide the following information so we can contact you.",
+          description: "Explanation message for request form.",
+        })}
+      </p>
+      <FormProvider {...methods}>
+        <form
+          onSubmit={handleSubmit(onSubmit)}
+          data-h2-flex-grid="b(top, contained, padded, none)"
+        >
+          <div
+            style={{ paddingRight: "1rem" }}
```

Comment:
This looks uneven when stacked. Media query for this (and line 172,`paddingLeft: "1rem"`) maybe?

![Screen Shot 2021-10-29 at 14 27 29](https://user-images.githubusercontent.com/3046459/139486029-54e4845a-2a28-4269-aac2-d20bd865f3f5.png)

Reply:
Yup..how did I miss that 😕

Reply:
I think you'll want to make [this](https://github.com/GCTC-NTGC/gc-digital-talent/blob/feature/request-page/talentsearch/resources/js/components/request/CreateRequest.tsx#L135) into a media query so that the padding is only applied when the inputs are not stacked.

![Screen Shot 2021-11-04 at 10 20 03](https://user-images.githubusercontent.com/3046459/140331826-a20e3917-5de8-4714-9f46-a37b29a84777.png)